index.js

```diff
@@ -305,6 +306,9 @@ type Mutation {
   toggleConditionOption(input: ToggleConditionOptionInput!): RoutingConditionValue
   toggleValidationRule(input: ToggleValidationRuleInput!): ValidationRule!
   updateValidationRule(input: UpdateValidationRuleInput!): ValidationRule!
+  createMetadata(input: CreateMetadataInput!): Metadata!
+  updateMetadata(input: UpdateMetadataInput!): Metadata!
+  deleteMetadata(input: DeleteMetadataInput!): Metadata
```

Comment:
Shouldn't the delete metadata return type be not-nullable as well, it should still return the pseudo-deleted object?

Reply:
Huh yeah I've just looked, apparently all our mutations besides validation rule return a nullable result. Not sure if that's the best thing as all mutations need to return something to update the cached store? Going forward it might be worth making everything not nullable and update the existing mutations some point in the future?